static/scripts/script.js

```diff
@@ -0,0 +1,67 @@
+var ws = new WebSocket("ws://localhost:8888/ws");
+
+ws.onmessage = function (evt) {
+    var json = JSON.parse(evt.data);
+
+    switch(json.label)
+    {
+        case "operator_text":
+        case "robot_text":
+            var subtitles = document.getElementById("subtitles");
+            var txt = document.createElement("li");
+            txt.appendChild(document.createTextNode(json.text));
+            txt.setAttribute("class", json.label + " subtitle-line");
+            console.log(json.label) // added line
+            subtitles.insertAdjacentElement('afterbegin', txt);
```

Comment:
The typical chat log appends at the bottom. 
Also clearing the current dummy-content the first time text is added would be nice too.

Reply:
I don't understand why you want to clear the dummy-content, for me it's just an example to see how messages will appear, there is no reason to keep it displayed during the cup. 
If it is mandatory to you, I can keep the dummy messages and remove them when receiving new text.

Reply:
No dummy text is fine too